### DISCUSSION TITLE
loxone light groups not always working well

### DISCUSSION THREAD

**Discussion by pereest** [JoDehli/PyLoxone#305] [Q&A]
Hello, I have in loxone groups defined for example living = spot 1,2,3,4,5,6 + central light
when I want to create scenes in home assistant, I have lots of problems, some of the lights sometimes do not switch state.
Are there some rules that I need to follow? For example define every light seperately in loxone??

**Comment by JoDehli** [comment]
@pereest I do not really know what you configuration is and what the problem is. Can you give me more Infos? Maybe some screenshots.
